packages/gestalt/src/SideNavigation/NavigationContent.js

```diff
@@ -0,0 +1,149 @@
+// @flow strict
+import { type Node as ReactNode, useEffect, useRef, useState } from 'react';
+import classnames from 'classnames';
+import Collapser from './Collapser';
+import {
+  countItemsWithIcon,
+  groupIconlessChildren,
+  validateChildren,
+} from './navigationChildrenUtils';
+import borderStyles from '../Borders.css';
+import Box from '../Box';
+import boxStyles from '../Box.css';
+import { useSideNavigation } from '../contexts/SideNavigationProvider';
+import Divider from '../Divider';
+import Flex from '../Flex';
+import layoutStyles from '../Layout.css';
+import { type Props as SideNavigationProps } from '../SideNavigation';
+import styles from '../SideNavigation.css';
+import { flattenChildrenWithKeys } from '../utils/flattenChildren';
+import { FixedZIndex } from '../zIndex';
+
+type Props = { ...SideNavigationProps };
+type Timeout = ReturnType<typeof setTimeout>;
+
+export default function NavigationContent({
+  accessibilityLabel,
+  children,
+  footer,
+  header,
+  showBorder,
+}: Props): ReactNode {
```

Comment:
why not just 

`}: SideNavigationProps): ReactNode {
`

?

`type Props = { ...SideNavigationProps };
`seems redundant, no?

Reply:
Sorry, I think I had some more additional props there but forgot to simplify after removing them.